video_chat_room.html

```diff
@@ -0,0 +1,18 @@
+<html>
```

Comment:
BTW, how come we have this page, instead of just sending them to meet.jit.si/whatever directly?

Reply:
It wasn't necessary but I did it for a few reasons.

1) It allows us to customize the page.  For example, I put the puzzle name at the top of the video chat which I thought might be a useful reminder of which video chat you are in.  
2) In the future, we needed this in case we ever want to use the API (to say put things in the messages, grab who is on video, use their czar name as their name on the video chat etc...) so I figured I would just do it.